packages/react-dev-utils/WebpackDevServerUtils.js

```diff
@@ -324,6 +324,10 @@ function prepareProxy(proxy, appPublicFolder) {
       // If this heuristic doesn’t work well for you, use a custom `proxy` object.
       context: function(pathname, req) {
         return (
+          (!pathname.startsWith("/sockjs-node/") &&
```

Comment:
```suggestion
          // Resolves an issue with WebSocket (#5280).
          (!pathname.startsWith("/sockjs-node/") &&
```

Reply:
@mrmckeb the socksjs part is there, because the proxy should not proxy the request from the auto reloading server to the other server. So I don't saw another solution to implement it by the path name.